src/types/events/message-events.ts

```diff
@@ -37,6 +37,7 @@ export interface GenericMessageEvent {
 export interface BotMessageEvent {
```

Comment:
thread_ts, channel_type are also missing

Reply:
`channel_type`'s type can be `"app_home" | "channel" | "group" | "im" | "mpim"` but I think just string is okay.

Reply:
I don't see `thread_ts` in either the Java data class, nor the generated JSON. Can you confirm it belongs here?

Reply:
@misscoded thanks for pointing this out. we need to add thread_ts in Java SDK too